packages/core/handle/handle.js

```diff
@@ -92,10 +92,20 @@ import {is} from '../keymap';
  * @callback HandlerFunction
  * @memberof core/handle
  * @param {Object<string, any>} event
- * @param {any} props
+ * @param {Object<string, any>} props
  * @param {Object<string, any>} context
  */
 
+ /**
+ * @callback EventAdapter
+ * @memberof core/handle
+ * @param {Object<string, any>} event
```

Comment:
I'm concerned this will be more restrictive than we ultimately want. While all the native events and all our events fulfill this type, we can't guarantee that all callbacks will which would force some type gymnastics on consumers for little value.

Reply:
I suspect, though, changing to `any` would mean that 100% of code will have to have a type override to access properties on events.  Would developers have to anyhow?

Reply:
They would have to anyway. TS will complain about implicit `any`s by default.

https://www.typescriptlang.org/play/index.html#code/FASwdgLgpgTgZgQwMZQAQFEBuVIAkFgAmANrKgN7CrWoAUVNjUmAXKgQJ4A0Dj1ADjAD2-AM5sASlCRCYhADyiIMcAHMu7MBwB8PPjRmQoADwiTpshUpVh1mnb1QBKNgCMhQ0gVQAfVJiEQQmAAX2BgQmliBBg0OABXMCQIECEwVAALAhI0WgA6AqyiUhhxDGw8bJKAbQBdF1R3TyhvPwCggG5whKSUtNRCIQBlCHi4ODpmNgBZIXjRKCwcCA1BETKpGTlFZTUNTm0nCkcQCdpmPNd4iAh+gF4H1ABGI8p9akNRZrziIVVaNZiPIpCCkJxdRhhMLAHrJVLpADuIAgGQAggA5eIAW1cZFoMkSZlQYGxuJgq2EYnMWysu1s+y0h2OjFOdAJkFQDzuz1ejkYn2+v3+gNEwORYIhNCh3UScP6BH6+LmkAp62plh2NjsB15LLO7IgnMeL2Z7wFpB+fwBlNFIIljmlwE+huMnMyVSgtAVYA0gxGYzgGiRKIxpNg4OAQA

If you toggle off the `noImplicitAny` option, the above will be free of errors.

Reply:
I guess it makes sense.  Particularly when talking about the `MouseEvent` case.  We aren't really taking advantage of the proper event types in any of the places where we could.